db/migrate/20160412142212_create_locations.rb

```diff
@@ -9,6 +9,6 @@ def change
     end
 
     add_foreign_key :flood_risk_engine_locations,
-                    :flood_risk_engine_addresses
+                    :flood_risk_engine_addresses, column: :address_id
   end
```

Comment:
@woodpigeon  - I think sqllite adaptor more flexible, using  Postgres without  that column def you get :

_PG::UndefinedColumn: ERROR:  column "flood_risk_engine_organisation_id" referenced in foreign key constraint does not exist
: ALTER TABLE "flood_risk_engine_enrollments" ADD CONSTRAINT "fk_rails_5f7f4a12fa"
FOREIGN KEY ("flood_risk_engine_organisation_id")
  REFERENCES "flood_risk_engine_organisations" ("id")_